### PR TITLE
add floating_tags and unique_tags to koji build metadata

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-7749

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by athos-ribeiro** [comment]
Waiving osbs-docs PR sine docs for koji metadata are included in this PR.

Since this is additional metadata for koji, I would also waive the new feature toggle requirement.

File: atomic_reactor/util.py [hunk 1]
```diff
@@ -1384,6 +1384,16 @@ def get_floating_images(workflow):
...
 
 
+def get_unique_images(workflow):
+    unique_images = workflow.tag_conf.unique_images
+    if not unique_images:
+        unique_results = workflow.build_result.annotations['repositories'].get('unique', [])
+        if unique_results:
+            unique_images = [ImageName.parse(unique) for unique in unique_results]
+
+    return unique_images
```
Comment (MartinBasti) [inline atomic_reactor/util.py]: Will be this empty list when no unique tags are defined?

Comment (rcerven) [reply atomic_reactor/util.py]: yeah

**Comment by rcerven** [comment]
rn updated: - floating_tags and unique_tags added to koji build metadata

**Pull request merged**